src/components/VencordSettings/ThemesTab.tsx

```diff
@@ -74,10 +92,193 @@ function Validators({ themeLinks }: { themeLinks: string[]; }) {
     );
 }
 
+interface ThemeCardProps {
+    theme: UserThemeHeader;
+    enabled: boolean;
+    onChange: (enabled: boolean) => void;
+    onDelete: () => void;
+}
+
+function ThemeCard({ theme, enabled, onChange, onDelete }: ThemeCardProps) {
+    function renderLinks() {
+        const links: (JSX.Element | string)[] = [];
```

Comment:
why not do this here instead of the `links.map<React.ReactNode>(t => t)` below?
```suggestion
        const links: React.ReactNode[] = [];
```